src/app/components/elements/modals/inequality/InequalityModal.tsx

```diff
@@ -425,60 +396,10 @@ const InequalityMenu = React.forwardRef<HTMLDivElement, InequalityMenuProps>(
-
-            {editorMode === "chemistry" && (
-              <>
-                {activeMenu === "elements" &&
-                  (isDefined(availableSymbols) && availableSymbols.length > 0 ? (
-                    <div className="top-menu chemistry elements">
-                      <ul className="sub-menu elements">{menuItems.chemicalElements.map(buildIndexedMenuItem)}</ul>
-                    </div>
-                  ) : (
-                    <div className="top-menu chemistry elements text-entry">
-                      <div className="input-box">
-                        <Input
-                          id="chem-text-entry-box"
-                          type="text"
-                          placeholder="Type chemical elements here"
-                          onMouseDown={(e) => e.currentTarget.focus()}
-                          value={unparsedChemicalElements || ""}
-                          onChange={onUnparsedChemicalElementsChange}
-                        />
-                        {upperCaseWarning && (
-                          <p className="uppercase-warning">
-                            Careful: chemical element names start with an upper-case letter.
-                          </p>
-                        )}
-                      </div>
-                      <div className="items-box">
-                        <ul className="sub-menu elements">{parsedChemicalElements?.map(buildIndexedMenuItem)}</ul>
-                      </div>
-                    </div>
-                  ))}
-                {activeMenu === "particles" &&
-                  (!isDefined(availableSymbols) || (isDefined(availableSymbols) && availableSymbols.length === 0)) && (
-                    <div className="top-menu chemistry particles">
-                      <ul className="sub-menu particles">{menuItems.chemicalParticles.map(buildIndexedMenuItem)}</ul>
-                    </div>
-                  )}
-                {activeMenu === "states" && (
-                  <div className="top-menu chemistry states">
-                    <ul className="sub-menu states">{menuItems.chemicalStates.map(buildIndexedMenuItem)}</ul>
-                  </div>
-                )}
-                {editorMode === "chemistry" && activeMenu === "operations" && (
-                  <div className="top-menu chemistry operations">
-                    <ul className="sub-menu operations">{menuItems.chemicalOperations.map(buildIndexedMenuItem)}</ul>
-                  </div>
-                )}
-              </>
-            )}
           </div>
           <div id="inequality-menu-tabs" className="menu-tabs">
             <ul>
-              {["maths", "chemistry"].includes(editorMode) && (
-                <InequalityMenuTab menu={"numbers"} latexTitle={"1\\, 2\\, 3"} />
-              )}
+              {editorMode === "maths" && <InequalityMenuTab menu={"numbers"} latexTitle={"1\\, 2\\, 3"} />}
               {["maths", "logic"].includes(editorMode) && (
```

Comment:
This statement is now always true. I'm not sure if it's worth changing, so it's up to you.

Reply:
Makes sense for clarity, so I updated before merging it in